core-spi-test/src/main/java/org/ehcache/internal/store/StoreComputeIfAbsentTest.java

```diff
@@ -33,10 +35,25 @@ public StoreComputeIfAbsentTest(StoreFactory<K, V> factory) {
     super(factory);
   }
 
+  protected Store<K, V> kvStore;
+  protected Store kvStore2;
```

Comment:
A raw store type is defined here as was being used in some tests to check behaviour in case of wrong key and/or value type. The definition has been moved out so a common tearDown method can close all stores at the end of each test